### ISSUE TITLE
Compile failed fdk-aac

### ISSUE THREAD

**Issue by rhadamants** [jb-alvarado/media-autobuild_suite#1166]
Compile fdk-aac but give error in git clone https://git.xiph.org/opus.git opus-git.

[compile.log](https://github.com/jb-alvarado/media-autobuild_suite/files/3077072/compile.log)
[compile.stripped.log](https://github.com/jb-alvarado/media-autobuild_suite/files/3077075/compile.stripped.log)

**Comment by 1480c1** [comment]
Rerun, timed out trying to connect

**Comment by rhadamants** [comment]
But site git.xiph.org/opus.git is fine.

**Comment by 1480c1** [comment]
Exactly, re run now since you have confirmed xiph is fine

**Comment by 1480c1** [comment]
from my personal experience, xiph is not always stable (ssl certs not being verified by wget, timing out sometimes, etc.)

**Comment by rhadamants** [comment]
Cloning into 'opus-git'. forever.
opus-git folder created and a .git folder too.
But continue cloning.

How to disable this opus and not affect fdk-aac.

**Comment by schmidthubert** [comment]
Why use the .org site anyway and why has that [been changed](https://github.com/jb-alvarado/media-autobuild_suite/commit/24b9ce00ed9c76babb7e1d3211e0c37d1220dbde) just for opus and away from github recently? Just github should be good, the script uses much more from xiph, so everything should be replaced with gitbub (again), IMO. I prefer "everything GH" is dead before "xyz is dead" or "xyz has problems" and have to search around for an alternative. If GH is dead sip a coffee and come back later! :+1: 

https://github.com/xiph

**Comment by 1480c1** [comment]
edit ffmpeg_options.txt under the build folder and remove --enable-libopus or put --disable-libopus

**Comment by 1480c1** [comment]
> Why use the .org site anyway and why has that [been changed](https://github.com/jb-alvarado/media-autobuild_suite/commit/24b9ce00ed9c76babb7e1d3211e0c37d1220dbde) just for opus and away from github recently? Just github should be good, the script uses much more from xiph, so everything should be replaced with github (again), IMO. I prefer "everything GH" is dead before "xyz is dead" or "xyz has problems" and have to search around for an alternative. If GH is dead sip a coffee and come back later! 👍
> 
> https://github.com/xiph

I do wonder why we moved to xiph.org links, since it seems that the commit hashes match between the two and there doesn't seem to be any difference other than the links themselves. \
@Helenerineium , why did we switch to using the xiph links? Was there a specific reason?

**Comment by wiiaboo** [comment]
Might be the same reason as FFmpeg and the github mirrors aren't automatically updated. But I forgot to ask why.

**Comment by 1480c1** [comment]
Hmm, maybe, but it does seem that the latest was already updated (9 hours ago for both for me). Will need to see how quickly github is updated

**Comment by rhadamants** [comment]
Compile in celeron G3930 is fine but i5 4590 give error.
Fist compile in G3930 is ok but give this error after:
> error: GPGME error: Invalid crypto engine

> If you think configure made a mistake, make sure you are using the latest
> version from Git.  If the latest version fails, report the problem to the
> ffmpeg-user@ffmpeg.org mailing list or IRC #ffmpeg on irc.freenode.net.
> Include the log file "ffbuild/config.log" produced by configure as this will help
> solve the problem.
> 
> /build/fail_comp: line 1: declare: `'': not a valid identifier
> /build/fail_comp: line 1: declare: `'': not a valid identifier
> /build/fail_comp: line 1: declare: `'': not a valid identifier
> /build/fail_comp: line 1: declare: `'': not a valid identifier
> This is required for other packages, so this script will exit.
> 
> All relevant logs have been anonymously uploaded to https://0x0.st/zNYM.zip
> Copy and paste [logs.zip](https://0x0.st/zNYM.zip) in the GitHub issue.
> Make sure the suite is up-to-date before reporting an issue. It might've been fixed already.
> Try running the build again at a later time.

**Comment by 1480c1** [comment]
That's #1168

**Comment by 1480c1** [comment]
Actually, that's something wrong with chromaprint
```
media-autobuild_suite-master/msys64/tmp/ffconf.OaoshX3T/test.c:1:10: fatal error: chromaprint.h: No such file or directory
 #include <chromaprint.h>
          ^~~~~~~~~~~~~~~
compilation terminated.
```

**Comment by Helenerineium** [comment]
> Actually, that's something wrong with chromaprint

> error: GPGME error: Invalid crypto engine

It is actually that gpg is broken so all the do_pacman_install attempts fail, and that's why chromaprint is not detected (it is supposed to be installed from pacman)

As for the xiph links, there are no exact reasons and I don't mind switching back.

**Comment by 1480c1** [comment]
> > Actually, that's something wrong with chromaprint
> 
> > error: GPGME error: Invalid crypto engine
> 
> It is actually that gpg is broken so all the do_pacman_install attempts fail, and that's why chromaprint is not detected (it is supposed to be installed from pacman)
> 
> As for the xiph links, there are no exact reasons and I don't mind switching back.

I would vote for using github mirrors for anything so long as the hashes etc are the same primarily because then there's less source sites we have to worry about working (github, sourceforge, notabug, etc)

**Comment by wiiaboo** [comment]
The notabug mirrors are to prevent relying on sourceforge, but if it's more reliable now, we can just use it directly.
I'd agree to move back to github for the xiph projects.
The FFmpeg github mirror is non-automatic, so I'd rather stay with the official repo.

Ideally, another low-priority feature I never bothered to implement are redundant mirrors for each package, which could also be implemented and would be make the suite more resiliant against censorship and downtime.

**Comment by 1480c1** [comment]
This can be closed by #1183
